package.json

```diff
@@ -17,11 +17,15 @@
     "env": "printenv",
     "clean:dist": "git checkout -- dist",
     "clean": "rm -rf generated dist lib .coverage && yarn run clean:dist",
-    "compile:node": "coffee --output lib --compile src",
-    "compile:test": "coffee --output generated/test --compile test",
+    "compile:node:coffee": "coffee --output lib --compile src",
```

Comment:
Sorry Jason, but yarn doesn't set the directories variables:

![screen shot 2017-03-09 at 8 30 37 am](https://cloud.githubusercontent.com/assets/79303/23741922/b5b7acc4-04a2-11e7-98cd-7396b6ba2d08.png)

Sounds like a drum for you to go beat over at @yarn